CogniteSdk/test/csharp/GZipTests.cs

```diff
@@ -0,0 +1,63 @@
+using System.IO;
```

Comment:
looks like we don't use "Test" as postfix in this folder. prob `GZip` or `GZipStream` is enough